src/test/suite/StartIntegration.test.ts

```diff
@@ -44,16 +46,18 @@ suite('Check can deploy default examples', () => {
 	let showInputBoxStub: sinon.SinonStub;
 	let showWorkspaceFolderPickStub: sinon.SinonStub;
 	let createdFile: vscode.Uri | undefined;
+	let telemetrySpy: sinon.SinonSpy;
 
-	setup(() => {
+	setup(async() => {
 		showQuickpickStub = sinon.stub(vscode.window, 'showQuickPick');
 		showInputBoxStub = sinon.stub(vscode.window, 'showInputBox');
 		showWorkspaceFolderPickStub = sinon.stub(vscode.window, 'showWorkspaceFolderPick');
 		// Workaround due to bug in shelljs: https://github.com/shelljs/shelljs/issues/704
 		shelljs.config.execPath = shelljs.which('node').toString();
+		telemetrySpy = sinon.spy(await getTelemetryServiceInstance(), 'send');
 	});
 
-	teardown(() => {
+	teardown(async() => {
```

Comment:
why is it required to make it async?

Reply:
I thougth it was needed as I switched the setup parameter method to async. And it is nto the case. I switch back to sync as it was before for the teardown which doesn't require it.